src/libmongoc/src/mongoc/mongoc-interrupt-private.h

```diff
@@ -23,8 +23,8 @@
 
 /* Creates a stream to use to interrupt calls to mongoc_stream_poll.
  *
- * The expected use is to cancel in-progress ismaster commands (especially for
- * awaitable ismaster). An ismaster command may not respond for a long time, so
+ * The expected use is to cancel in-progress hello commands (especially for
+ * awaitable hello). An hello command may not respond for a long time, so
```

Comment:
```suggestion
 * awaitable hello). A hello command may not respond for a long time, so
```

Reply:
Did a pass through the code to catch all instances of "an hello command"